### ISSUE TITLE
Config for dragon flight speed

### ISSUE THREAD

**Issue by Spinogaming** [AlexModGuy/Ice-and-Fire-Suggestions#615]
There should be a config for the flight speed of dragons, mainly because dragons like to fly faster than my chunks can load

**Comment by Drakeofdark** [comment]
YES, I can't play 1.8.4 just because of how unbearable flight is and the fact that it makes your armor invisible for some reason
